packages/web/src/scss/tools/_themes.scss

```
@@ -62,13 +62,22 @@
+            url('data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 20 20" fill="none"><path fill="#{map.get($themed-tokens, colors, action, toggle, unselected-content)}" fill-rule="evenodd" d="M20 0H0v20h20V0Zm-7.302 6.177a.58.58 0 0 1 .823 0 .585.585 0 0 1 0 .816l-2.853 2.853 2.853 2.852a.58.58 0 1 1-.823.823l-2.852-2.853-2.853 2.853a.58.58 0 1 1-.822-.823l2.852-2.852-2.852-2.853a.58.58 0 1 1 .822-.822l2.853 2.852 2.852-2.846Z" clip-rule="evenodd"/></svg>')
+        )};
+    --#{globals.$prefix}toggle-input-mark-indicators-disabled-background-image-url: #{svg.escape(
+            url('data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 20 20" fill="none"><path fill="#{map.get($themed-tokens, colors, disabled, foreground)}" fill-rule="evenodd" d="M20 0H0v20h20V0Zm-7.302 6.177a.58.58 0 0 1 .823 0 .585.585 0 0 1 0 .816l-2.853 2.853 2.853 2.852a.58.58 0 1 1-.823.823l-2.852-2.853-2.853 2.853a.58.58 0 1 1-.822-.823l2.852-2.852-2.852-2.853a.58.58 0 1 1 .822-.822l2.853 2.852 2.852-2.846Z" clip-rule="evenodd"/></svg>')
+        )};
+    --#{globals.$prefix}toggle-input-mark-indicators-checked-background-image-url: #{svg.escape(
+            url('data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 20 20" fill="none"><path fill="#{map.get($themed-tokens, colors, action, toggle, selected-content)}" fill-rule="evenodd" d="M20 0H0v20h20V0ZM5.987 9.904l2.042 2.042 5.775-5.775a.574.574 0 0 1 .817 0 .574.574 0 0 1 0 .816l-6.184 6.178a.58.58 0 0 1-.822 0L5.17 10.72a.574.574 0 0 1 0-.817.574.574 0 0 1 .816 0Z" clip-rule="evenodd"/></svg>')
+        )};
+    --#{globals.$prefix}toggle-input-mark-indicators-disabled-checked-background-image-url: #{svg.escape(
```

I'd swap the two words, but it's just a tiny detail…

```suggestion
    --#{globals.$prefix}toggle-input-mark-indicators-checked-disabled-background-image-url: #{svg.escape(
```